server/src/main/java/com/uj/projects/booksplatform/category/controller/CategoryController.java

```diff
@@ -0,0 +1,29 @@
+package com.uj.projects.booksplatform.category.controller;
+
+import com.uj.projects.booksplatform.category.entity.Category;
+import com.uj.projects.booksplatform.category.service.CategoryService;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+
+    private CategoryService categoryService;
+
+    public CategoryController(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
+
+    @GetMapping
+    public List<Category> getAll(){
+        return categoryService.getAll();
+    }
+
```

Comment:
Shouldn't function also has @ CrossOrigin(origins = "*", allowedHeaders = "*") like PostMapping in UserController ?

Reply:
we enabled this globally. You cannot see it, because it is on destination branch